### ISSUE TITLE
ContentType Negotiation Required

### ISSUE THREAD

**Issue by mav2287** [middlewares/error-response#1]
Went to use this without knowing there had to be a Content-type negotiation. Had to go into the source to figure that out. Might be something use to add into the docs since it won't work out of the box without it.

**Comment by oscarotero** [comment]
Yes, you're right, the default behavior needs content-type header in the response. You can use the [Negotiation](https://github.com/middlewares/negotiation) middleware for that.

But you can create your own error response without check it, just have to implement the [ResponderInterface](https://github.com/middlewares/error-response/blob/master/src/ErrorResponder/ResponderInterface.php) and returning `true` for the `isValid()` method. And maybe there could be a generic responder to be used as fallback if not valid content type has found.
